openapi.yaml

```diff
@@ -12463,6 +12465,45 @@ components:
                 a relative URL, it is relative to the domain you retrieved the
                 Event from.
               example: /v4/support/tickets/11111
+        secondary_entity:
+          type: object
+          readOnly: true
+          description: |
+            Detailed information about the Event's secondary entity, which provides additional information
+            for `linode_boot`, `linode_create`, and `linode_clone` Event actions.
+          x-linode-cli-display: 5
+          properties:
+            id:
+              type: integer
+              description: |
+                The unique ID for an Event's secondary entity.
+
+
+                Secondary Entities are assigned the ID of the Linode they correspond to.
+              example: 1234
+            label:
+              type: string
+              description: >
+                The current label of this object. The label may reflect changes
+                that occur with this Event.
```

Comment:
Actually no; secondary entity labels won't change.  Entity labels may stop being updated in the future, also.